nominees/sormas.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "SORMAS",
```

Comment:
@ijeomaemeruwa could you add an alias field and borrow a leaf from the [DHIS2](https://github.com/unicef/publicgoods-candidates/blob/master/nominees/dhis2.json) submission. Thanks.